### ISSUE TITLE
4a-WVD-scaling-logicapps - Select-azSubscription : Please provide a valid tenant or a valid subscription

### ISSUE THREAD

**Issue by chiners68** [Azure/RDS-Templates#472]
I Started to run through the scaling script but bombed out as I couldn't find the URI. Started the process again now evetime I create the Automation account with the command below (subscription ID removed) I get the ID is invalid. The Subscription ID is valid I've verified it 100 times, and it worked the first time before I exited out to start again.

.\createazureautomationaccount.ps1 -SubscriptionID <Subscription ID>  -ResourceGroupName AA-UKS-RG-WVD1 -AutomationAccountName AA-WVD-UKS -Location "UKSouth"

Error
Select-azSubscription : Please provide a valid tenant or a valid subscription.
At C:\temp\createazureautomationaccount.ps1:85 char:17
+ $Subscription = Select-azSubscription -SubscriptionId $SubscriptionId
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Set-AzContext], ArgumentException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.Profile.SetAzureRMContextCommand

**Comment by YariCa** [comment]
Hi, from my side, solved executing first

login-azaccount
then execute the script.
Yari

**Comment by chiners68** [comment]
I have executed the azure login before each attempt, I have also tried different accounts to no avail.

**Comment by chiners68** [comment]
Sorted.
I done a connect-AZaccount for something else I was setting up, but what ever account I connected with it used an account with no permissions. So I done a Disconnect-AzAccount before login-AZaccount and it worked. I repeated this before attemmpting the scaling script and now I can see the subscription ID.

how it ever managed to use the random account it was actually using is a mystery to me.
